[2024-03-01T16:42:39.896+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T11:10:00+00:00 [queued]>
[2024-03-01T16:42:39.904+0530] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T11:10:00+00:00 [queued]>
[2024-03-01T16:42:39.905+0530] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-03-01T16:42:40.061+0530] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): zipallfiles_and_placeatrequiredlocation> on 2024-03-01 11:10:00+00:00
[2024-03-01T16:42:40.099+0530] {standard_task_runner.py:60} INFO - Started process 23818 to run task
[2024-03-01T16:42:40.105+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'datapipeline_Take4', 'zipallfiles_and_placeatrequiredlocation', 'scheduled__2024-03-01T11:10:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/datapipeline_best(2).py', '--cfg-path', '/tmp/tmp3ytgl277']
[2024-03-01T16:42:40.108+0530] {standard_task_runner.py:88} INFO - Job 148: Subtask zipallfiles_and_placeatrequiredlocation
[2024-03-01T16:42:40.279+0530] {task_command.py:423} INFO - Running <TaskInstance: datapipeline_Take4.zipallfiles_and_placeatrequiredlocation scheduled__2024-03-01T11:10:00+00:00 [running]> on host arun-VM
[2024-03-01T16:42:40.441+0530] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='datapipeline_Take4' AIRFLOW_CTX_TASK_ID='zipallfiles_and_placeatrequiredlocation' AIRFLOW_CTX_EXECUTION_DATE='2024-03-01T11:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-01T11:10:00+00:00'
[2024-03-01T16:42:40.465+0530] {python.py:201} INFO - Done. Returned value was: None
[2024-03-01T16:42:40.477+0530] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=datapipeline_Take4, task_id=zipallfiles_and_placeatrequiredlocation, execution_date=20240301T111000, start_date=20240301T111239, end_date=20240301T111240
[2024-03-01T16:42:40.544+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-01T16:42:40.558+0530] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
